tests/test_basic/results.py

```diff
@@ -3,14 +3,13 @@
 import sys
 
 # import statepoint
-sys.path.insert(0, '../../src/utils')
```

Comment:
Ok, I set this up assuming that the user had installed the API along with the code before running the tests. I'll add the ``sys.path.insert(...)`` calls back to all of the tests to avoid this issue.